### ISSUE TITLE
Extra square brackets around citations in figure captions

### ISSUE THREAD

**Issue by dhimmel** [jgm/pandoc#4272]
In 2.1 colons are added around every citation in figure legends, in addition to colons for the entire span of citations.

Before the issue ([HTML page](https://htmlpreview.github.io/?https://github.com/greenelab/scihub-manuscript/blob/cc8212b85944392fe9f1e1dbc5a069550a2c95bc/index.html)):

![before](https://user-images.githubusercontent.com/1117703/35010506-14deadcc-fad1-11e7-8a9c-49a9676a4836.png)

After the issue ([HTML page](https://htmlpreview.github.io/?https://github.com/greenelab/scihub-manuscript/blob/1c32e7760ea4c73b00f95129213dcc3d82f627b8/index.html)):

![after](https://user-images.githubusercontent.com/1117703/35010509-169f1eb2-fad1-11e7-876c-6a559487303c.png)

Here's the [markdown source](https://github.com/greenelab/scihub-manuscript/blob/1a89630d98ed82949487d4f167f23fa6a1f34cc3/manuscript.md):

```md
![
**The history of Sci-Hub.**
Weekly interest from Google Trends is plotted over time for the search terms "Sci-Hub" and "LibGen".
The light green period indicates when Sci-Hub used LibGen as its database for storing articles [@1H8x8f4Y7].
The light blue period indicates the public availability of access logs from Sci-Hub [@kQFQ8EaO].
The first pink dotted line represents the collection date of the LibGen metadata used in Cabanac's study [@18KKRwlN6; @GxWkjeN2].
The second pink dotted line shows the date of Sci-Hub's Tweeted DOI catalog used in this study.
](https://cdn.rawgit.com/greenelab/scihub/02851bfad35f08ab4d64b9b5a671eebac951cdb9/explore/trends/google-trends.png){#fig:history width="100%"}
```

Here's the [pandoc command](https://github.com/greenelab/scihub-manuscript/blob/5cb1245ca3e68c425b1364317e2496c17e20353c/build/build.sh#L28-L43).

This issue does not occur with pandoc 2.0.5 but does occur with 2.1.0. It affects figure captions (i.e. markdown's alt text in image embed) but does not seem to effect table captions (i.e. `Table: caption_here` that follows the table).

**Comment by jgm** [comment]
This should be fixed by commit d9584d73f94501787026c57b77d217e51f21505d

**Comment by dhimmel** [comment]
Nice. Don't know what I was thinking calling `[` a colon haha
